### DISCUSSION TITLE
How to create keymap to cycle through the list

### DISCUSSION THREAD

**Discussion by chuan137** [ibhagwan/fzf-lua#1202] [General]
use <c-j> or <c-n> to go down the list, and at the last line it does not do anything, because there is no next item. How to make them automatically jump to the top?

**Comment by ibhagwan** [comment]
Add `['--cycle']=true` to your `fzf_opts`, you can test it with:
```lua
:FzfLua files fzf_opts.--cycle=true
```

**Comment by ibhagwan** [comment]
Searching for “cycle” in issues: https://github.com/ibhagwan/fzf-lua/issues/442
